src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -91,9 +91,46 @@
 }
 
 .actions {
-  @apply items-stretch flex justify-end m-0 flex-initial;
+  @apply items-stretch flex m-0 flex-initial;
 }
 
 .actions--start ~ .label {
+  @apply pl-1;
```

Comment:
Can you use `padding-inline-start` instead of left? Then we can remove the padding-right and RTL classes.

Reply:
This styling was a change that was included when I merged master into this branch. I didn't want to alter it and accepted this current change when resolving merge conflicts